invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/UppyUploader/RDMUppyUploaderPlugin.js

```diff
@@ -0,0 +1,417 @@
+// This file is part of Invenio-RDM-Records
+// Copyright (C) 2020-2025 CERN.
+// Copyright (C)      2025 CESNET.
+//
+// Invenio-RDM-Records is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+// NOTE: If WASM undesirable, we can use library like Crypto-JS
+import { md5 } from "hash-wasm";
+import AwsS3Multipart from "@uppy/aws-s3-multipart";
+import { fetcher } from "@uppy/utils/lib/fetcher";
+import { humanReadableBytes } from "react-invenio-forms";
+
+import { FileSizeError, InvalidPartNumberError, SignedUrlExpiredError } from "./error";
+
+const defaultOptions = {
+  // NOTE: null here means “include all”, [] means include none.
+  // This is inconsistent with @uppy/aws-s3 and @uppy/transloadit
+  allowedMetaFields: null,
+  limit: 3,
+  getTemporarySecurityCredentials: false,
+  getUploadParameters: null,
+  shouldUseMultipart: null,
+  uploadPartBytes: null,
+  retryDelays: [0, 1000, 3000, 5000],
+  companionHeaders: {},
+  checkPartIntegrity: false,
+};
+
+export class RDMUppyUploaderPlugin extends AwsS3Multipart {
+  static VERSION = "1.0.0";
```

Comment:
how is this variable used?

Reply:
It's just plugin class metadata, not very useful, I think it could be removed without any harm

Reply:
Now it's gone :)